tests/features/test_audio.py

```diff
@@ -126,6 +146,18 @@ def test_audio_decode_example_opus(shared_datadir):
     assert decoded_example["sampling_rate"] == 48000
 
 
+def test_audio_decode_example_pcm(shared_datadir):
+    sampling_rate = 16_000
```

Comment:
Maybe this test can make sure decoding uses the sampling rate of Audio(), not the sampling rate of the input example dictionary

```suggestion
@pytest.mark.parametrize("sampling_rate", [16_000, 48_000])
def test_audio_decode_example_pcm(shared_datadir, sampling_rate):
```